WooCommerce/Classes/ViewRelated/ReusableViews/EmptyStateViewController/EmptyStateViewController.swift

```diff
@@ -177,6 +177,7 @@ final class EmptyStateViewController: UIViewController, KeyboardFrameAdjustmentP
     /// Configure the elements to be displayed.
     ///
     func configure(_ config: Config) {
+        _ = view // trigger loading view before configuring contents
```

Comment:
just wondering, what are the scenarios this trigger is needed?

Reply:
This happens when I try to set config to the empty view (`emptyStateViewController.configure(config)`). We never got a crash before because in other cases we tend to display the view controller before configuring it, which triggers loading the view. In my case, I can't display the view right away so this workaround helps remedy that 😅

Reply:
But please suggest another solution if you think it might be better, thanks!